Content migration from "Restore Accounts" to "Account Generation" (#4062)
w3f/polkadot-wiki: docs/learn/learn-account-generation.md

:::

> ### Set Address for Polkadot Mainnet

Reviewer: ```suggestion
### Account Address Format
```

> In Polkadot you can use the same address on multiple chains. To switch between chains you can follow

Reviewer: ```suggestion
An account created for Polkadot can also be used on multiple chains in the Polkadot ecosystem. More specifically, the account of a chain that uses the `*25519` account address format (the latest list can be accessed on the [ss58 registry repository](https://github.com/paritytech/ss58-registry/blob/main/ss58-registry.json)) is cross-compatible with all the chains that use the similar format. To switch between the accounts on different chains, you can follow
```

the guidelines in
[this support article](https://support.polkadot.network/support/solutions/articles/65000103707-can-i-use-the-same-account-on-polkadot-kusama-and-parachains-).

> The [Accounts page](learn-accounts.md#address-conversion-tools) has a tool you can use to convert

Reviewer: The link does not point to the address conversion tools section. I tried searching for it in the learn-acocunts, but could not find it.

Author: fixed, ref to subscan conversion tool

your address between the different chain formats.

:::info For privacy reasons, we recommend creating a new address for each chain you're using.

Your address' format is only visual - the data used to derive this representation of your address
are the same, so you can use the same address on multiple chains. However, for privacy reasons, we
recommend creating a new address for each chain you're using.

> :::

Reviewer: This recommendation might need re-phrasing. Privacy may not be a concern for some account holders on Polkadot and Kusama, as many already have on-chain identity. It may be a good idea to elaborate on pros and cons, and let the Wiki reader make a judgement on whether to choose a single account on multiple chains or different accounts for each chain. 

```suggestion
:::info Using the same account on multiple chains - Pros and Cons

The address format differs from chain to chain, but that difference is only visual. The same private key can be used to sign transactions on behalf of the respective accounts on multiple chains. Using a single account on multiple chains is convenient, as you do not have to deal with multiple mnemonic phrases or private keys. But, if your account gets compromised on one chain, the attacker can gain full access to the accounts on all other chains. This also has implications for the account holder's privacy, as knowing the identity of an account on one chain can expose the account holder's identity on all the chains.

:::
```

> 

Reviewer: Let's add a screenshot of the account icon here

> You can copy your address by clicking on the account's icon while the desired chain format is

Reviewer: ```suggestion
On Polkadot-JS Extension, you can copy your address by clicking on the account's icon while the desired chain format is
```
